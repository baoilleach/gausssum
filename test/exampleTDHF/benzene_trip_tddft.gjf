%chk=benzene_trip.chk
# UHF/3-21G guess=read geom=allcheck td