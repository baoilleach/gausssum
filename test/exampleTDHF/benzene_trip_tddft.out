 Entering Link 1 = C:\G03W\l1.exe PID=       980.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2004,2007, Gaussian, Inc.
                  All Rights Reserved.
  
 This is the Gaussian(R) 03 program.  It is based on the
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 03, Revision E.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, J. A. Montgomery, Jr., T. Vreven, 
 K. N. Kudin, J. C. Burant, J. M. Millam, S. S. Iyengar, J. Tomasi, 
 V. Barone, B. Mennucci, M. Cossi, G. Scalmani, N. Rega, 
 G. A. Petersson, H. Nakatsuji, M. Hada, M. Ehara, K. Toyota, 
 R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, 
 H. Nakai, M. Klene, X. Li, J. E. Knox, H. P. Hratchian, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 P. Y. Ayala, K. Morokuma, G. A. Voth, P. Salvador, J. J. Dannenberg, 
 V. G. Zakrzewski, S. Dapprich, A. D. Daniels, M. C. Strain, 
 O. Farkas, D. K. Malick, A. D. Rabuck, K. Raghavachari, 
 J. B. Foresman, J. V. Ortiz, Q. Cui, A. G. Baboul, S. Clifford, 
 J. Cioslowski, B. B. Stefanov, G. Liu, A. Liashenko, P. Piskorz, 
 I. Komaromi, R. L. Martin, D. J. Fox, T. Keith, M. A. Al-Laham, 
 C. Y. Peng, A. Nanayakkara, M. Challacombe, P. M. W. Gill, 
 B. Johnson, W. Chen, M. W. Wong, C. Gonzalez, and J. A. Pople, 
 Gaussian, Inc., Wallingford CT, 2004.
 
 ******************************************
 Gaussian 03:  IA32W-G03RevE.01 11-Sep-2007
                20-May-2009 
 ******************************************
 %chk=benzene_trip.chk
 ---------------------------------------
 # UHF/3-21G guess=read geom=allcheck td
 ---------------------------------------
 1/29=7,38=1/1;
 2/40=1/2;
 3/5=5,11=2,16=1,25=1,30=1/1,2,8,3;
 4/5=1,7=2/1;
 5/5=2,38=6/2;
 8/6=1,10=1/1;
 9/42=1,70=1/14;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 --------
 _Benzene
 --------
 Redundant internal coordinates taken from checkpoint file:
 benzene_trip.chk
 Charge =  1 Multiplicity = 2
 H,0,0.,0.,-0.0113337391
 C,0,0.,0.,1.0581929106
 C,0,1.1879227257,-0.0000000001,1.7618895135
 H,0,2.1278341857,0.,1.2473173325
 C,0,1.1879227257,0.,3.2097104865
 H,0,2.1278341857,0.,3.7242826675
 C,0,0.,-0.0000000001,3.9134070894
 H,0,0.,0.,4.9829337391
 C,0,-1.1879227257,0.0000000001,3.2097104865
 H,0,-2.1278341857,0.,3.7242826675
 C,0,-1.1879227257,0.,1.7618895135
 H,0,-2.1278341857,0.,1.2473173325
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic     Atomic              Coordinates (Angstroms)
 Number     Number      Type              X           Y           Z
 ---------------------------------------------------------------------
    1          1             0        0.000000    0.000000   -0.011334
    2          6             0        0.000000    0.000000    1.058193
    3          6             0        1.187923    0.000000    1.761890
    4          1             0        2.127834    0.000000    1.247317
    5          6             0        1.187923    0.000000    3.209710
    6          1             0        2.127834    0.000000    3.724283
    7          6             0        0.000000    0.000000    3.913407
    8          1             0        0.000000    0.000000    4.982934
    9          6             0       -1.187923    0.000000    3.209710
   10          1             0       -2.127834    0.000000    3.724283
   11          6             0       -1.187923    0.000000    1.761890
   12          1             0       -2.127834    0.000000    1.247317
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  H    0.000000
     2  C    1.069527   0.000000
     3  C    2.134357   1.380706   0.000000
     4  H    2.472222   2.136222   1.071549   0.000000
     5  C    3.433116   2.457679   1.447821   2.175872   0.000000
     6  H    4.299129   3.411116   2.175872   2.476965   1.071549
     7  C    3.924741   2.855214   2.457679   3.411116   1.380706
     8  H    4.994267   3.924741   3.433116   4.299129   2.134357
     9  C    3.433116   2.457679   2.782234   3.852951   2.375845
    10  H    4.299129   3.411116   3.852951   4.924030   3.355448
    11  C    2.134357   1.380706   2.375845   3.355448   2.782234
    12  H    2.472222   2.136222   3.355448   4.255668   3.852951
                    6          7          8          9         10
     6  H    0.000000
     7  C    2.136222   0.000000
     8  H    2.472222   1.069527   0.000000
     9  C    3.355448   1.380706   2.134357   0.000000
    10  H    4.255668   2.136222   2.472222   1.071549   0.000000
    11  C    3.852951   2.457679   3.433116   1.447821   2.175872
    12  H    4.924030   3.411116   4.299129   2.175872   2.476965
                   11         12
    11  C    0.000000
    12  H    1.071549   0.000000
 Stoichiometry    C6H6(1+,2)
 Framework group  D2H[C2"(HC.CH),SG(C4H4)]
 Deg. of freedom     6
 Full point group                 D2H
 Largest Abelian subgroup         D2H     NOp   8
 Largest concise Abelian subgroup D2      NOp   4
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic     Atomic              Coordinates (Angstroms)
 Number     Number      Type              X           Y           Z
 ---------------------------------------------------------------------
    1          1             0        0.000000    0.000000    2.497134
    2          6             0        0.000000    0.000000    1.427607
    3          6             0        0.000000    1.187923    0.723910
    4          1             0        0.000000    2.127834    1.238483
    5          6             0        0.000000    1.187923   -0.723910
    6          1             0        0.000000    2.127834   -1.238483
    7          6             0        0.000000    0.000000   -1.427607
    8          1             0        0.000000    0.000000   -2.497134
    9          6             0        0.000000   -1.187923   -0.723910
   10          1             0        0.000000   -2.127834   -1.238483
   11          6             0        0.000000   -1.187923    0.723910
   12          1             0        0.000000   -2.127834    1.238483
 ---------------------------------------------------------------------
 Rotational constants (GHZ):      5.8773113      5.4447196      2.8263756
 Standard basis: 3-21G (6D, 7F)
 There are    16 symmetry adapted basis functions of AG  symmetry.
 There are     2 symmetry adapted basis functions of B1G symmetry.
 There are     4 symmetry adapted basis functions of B2G symmetry.
 There are    11 symmetry adapted basis functions of B3G symmetry.
 There are     2 symmetry adapted basis functions of AU  symmetry.
 There are    16 symmetry adapted basis functions of B1U symmetry.
 There are    11 symmetry adapted basis functions of B2U symmetry.
 There are     4 symmetry adapted basis functions of B3U symmetry.
 Integral buffers will be    262144 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
    66 basis functions,   108 primitive gaussians,    66 cartesian basis functions
    21 alpha electrons       20 beta electrons
       nuclear repulsion energy       202.9039403459 Hartrees.
 NAtoms=   12 NActive=   12 NUniq=    4 SFac= 3.00D+00 NAtFMM=   80 NAOKFM=F Big=F
 One-electron integrals computed using PRISM.
 NBasis=    66 RedAO= T  NBF=    16     2     4    11     2    16    11     4
 NBsUse=    66 1.00D-06 NBFU=    16     2     4    11     2    16    11     4
 Initial guess read from the checkpoint file:
 benzene_trip.chk
 Initial guess orbital symmetries:
 Alpha Orbitals:
       Occupied  (B1U) (B3G) (B2U) (AG) (B1U) (AG) (AG) (B2U) (B1U)
                 (B3G) (AG) (AG) (B1U) (B2U) (B2U) (B1U) (B3U)
                 (B3G) (AG) (B1G) (B2G)
       Virtual   (AU) (B3U) (AG) (B2G) (B2U) (B1U) (B3G) (B1U)
                 (AG) (B3G) (B1U) (AG) (B2U) (B1U) (B3G) (B2U)
                 (AG) (B3U) (B3G) (AG) (B1G) (B2G) (B1U) (AU) (B2U)
                 (B3U) (AG) (B2G) (B1U) (B3G) (AG) (B3G) (B2U)
                 (B1U) (B1U) (B2U) (AG) (B3G) (B1U) (AG) (B1U)
                 (B2U) (B3G) (AG) (B1U)
 Beta  Orbitals:
       Occupied  (B1U) (B3G) (B2U) (AG) (B1U) (AG) (AG) (B1U) (B2U)
                 (B3G) (AG) (AG) (B1U) (B2U) (B1U) (B2U) (B3G)
                 (AG) (B3U) (B2G)
       Virtual   (B1G) (B3U) (AU) (AG) (B2U) (B1U) (B2G) (B3G)
                 (AG) (B1U) (B3G) (B1U) (AG) (B2U) (B3G) (B1U)
                 (B2U) (AG) (B3U) (B3G) (AG) (B2G) (B1U) (B1G)
                 (B2U) (B3U) (AU) (AG) (B2G) (B1U) (B3G) (AG) (B3G)
                 (B2U) (B1U) (B1U) (B2U) (AG) (B3G) (B1U) (AG)
                 (B1U) (B2U) (B3G) (AG) (B1U)
 <S**2> of initial guess= 0.8598
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 Keep R1 and R2 integrals in memory in canonical form, NReq=     5871078.
 SCF Done:  E(UHF) =  -229.127423656     A.U. after    1 cycles
             Convg  =    0.2088D-09             -V/T =  2.0020
             S**2   =   0.8598
 Annihilation of the first spin contaminant:
 S**2 before annihilation     0.8598,   after     0.7568
 ExpMin= 1.83D-01 ExpMax= 1.72D+02 ExpMxC= 1.72D+02 IAcc=1 IRadAn=         1 AccDes= 1.00D-06
 HarFok:  IExCor= 205 AccDes= 1.00D-06 IRadAn=         1 IDoV=1
 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     7    66
 NBasis=    66 NAE=    21 NBE=    20 NFC=     6 NFV=     0
 NROrb=     60 NOA=    15 NOB=    14 NVA=    45 NVB=    46
 Would need an additional     1844642 words for in-memory AO integral storage.
 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (B1U) (B3G) (B2U) (AG) (B1U) (AG) (AG) (B2U) (B1U)
                 (B3G) (AG) (AG) (B1U) (B2U) (B2U) (B1U) (B3U)
                 (B3G) (AG) (B1G) (B2G)
       Virtual   (AU) (B3U) (AG) (B2G) (B2U) (B1U) (B3G) (B1U)
                 (AG) (B3G) (B1U) (AG) (B2U) (B1U) (B3G) (B2U)
                 (AG) (B3U) (B3G) (AG) (B1G) (B2G) (B1U) (AU) (B2U)
                 (B3U) (AG) (B2G) (B1U) (B3G) (AG) (B3G) (B2U)
                 (B1U) (B1U) (B2U) (AG) (B3G) (B1U) (AG) (B1U)
                 (B2U) (B3G) (AG) (B1U)
 Beta  Orbitals:
       Occupied  (B1U) (B3G) (B2U) (AG) (B1U) (AG) (AG) (B1U) (B2U)
                 (B3G) (AG) (AG) (B1U) (B2U) (B1U) (B2U) (B3G)
                 (AG) (B3U) (B2G)
       Virtual   (B1G) (B3U) (AU) (AG) (B2U) (B1U) (B2G) (B3G)
                 (AG) (B1U) (B3G) (B1U) (AG) (B2U) (B3G) (B1U)
                 (B2U) (AG) (B3U) (B3G) (AG) (B2G) (B1U) (B1G)
                 (B2U) (B3U) (AU) (AG) (B2G) (B1U) (B3G) (AG) (B3G)
                 (B2U) (B1U) (B1U) (B2U) (AG) (B3G) (B1U) (AG)
                 (B1U) (B2U) (B3G) (AG) (B1U)
    24 initial guesses have been made.
 Convergence on wavefunction:    0.001000000000000
 Iteration     1 Dimension    24 NMult    24
 CISAX will form  24 AO SS matrices at one time.
 No map to state      3
 Iteration     2 Dimension    30 NMult    30
 Iteration     3 Dimension    36 NMult    36
 Iteration     4 Dimension    42 NMult    42
 Iteration     5 Dimension    48 NMult    48
 Iteration     6 Dimension    52 NMult    52
 ***********************************************************************
 Excited states from <AA,BB:AA,BB> singles matrix:
 ***********************************************************************
 
 Ground to excited state Transition electric dipole moments (Au):
       state          X           Y           Z          Osc.
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.2301      0.0000      0.0038
         3         0.0000      0.0000     -0.5175      0.0216
 Ground to excited state transition velocity dipole Moments (Au):
       state          X           Y           Z          Osc.
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000     -0.0154      0.0000      0.0014
         3         0.0000      0.0000      0.0455      0.0114
 Ground to excited state transition magnetic dipole Moments (Au):
       state          X           Y           Z
         1        -0.7584      0.0000      0.0000
         2         0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000
 <0|del|b> * <b|rxdel|0>  (Au), Rotatory Strengths (R) in
 cgs (10**-40 erg-esu-cm/Gauss)
       state          X           Y           Z     R(velocity)
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000
 <0|r|b> * <b|rxdel|0>  (Au), Rotatory Strengths (R) in
 cgs (10**-40 erg-esu-cm/Gauss)
       state          X           Y           Z     R(length)
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000
  <0|del|b> * <b|r|0>  (Au)
       state          X           Y           Z       Osc.(frdel)
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000     -0.0035      0.0000      0.0024
         3         0.0000      0.0000     -0.0235      0.0157
 Ground to excited state transition densities written to RWF  633

 Excitation energies and oscillator strengths:
 
 Excited State   1:   ?Spin  -B3G    1.5986 eV  775.58 nm  f=0.0000
     17A -> 22A       -0.25418
     20A -> 25A        0.15565
     19B -> 23B        0.31252
     20B -> 21B        1.18760
 This state for optimization and/or second-order correction.
 Copying the excited state density for this state as the 1-particle RhoCI density.
 
 Excited State   2:   ?Spin  -B2U    2.9609 eV  418.74 nm  f=0.0038
     20A -> 23A       -0.10720
     21A -> 22A        0.93018
     19B -> 21B       -0.89185
     20B -> 23B       -0.37245
 
 Excited State   3:   ?Spin  -B1U    3.2974 eV  376.00 nm  f=0.0216
     17A -> 25A       -0.34708
     20A -> 22A        0.89330
     21A -> 23A       -0.95094
     19B -> 27B       -0.28691
     20B -> 22B        0.59036

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (B1U) (B3G) (B2U) (AG) (B1U) (AG) (AG) (B2U) (B1U)
                 (B3G) (AG) (AG) (B1U) (B2U) (B2U) (B1U) (B3U)
                 (B3G) (AG) (B1G) (B2G)
       Virtual   (AU) (B3U) (AG) (B2G) (B2U) (B1U) (B3G) (B1U)
                 (AG) (B3G) (B1U) (AG) (B2U) (B1U) (B3G) (B2U)
                 (AG) (B3U) (B3G) (AG) (B1G) (B2G) (B1U) (AU) (B2U)
                 (B3U) (AG) (B2G) (B1U) (B3G) (AG) (B3G) (B2U)
                 (B1U) (B1U) (B2U) (AG) (B3G) (B1U) (AG) (B1U)
                 (B2U) (B3G) (AG) (B1U)
 Beta  Orbitals:
       Occupied  (B1U) (B3G) (B2U) (AG) (B1U) (AG) (AG) (B1U) (B2U)
                 (B3G) (AG) (AG) (B1U) (B2U) (B1U) (B2U) (B3G)
                 (AG) (B3U) (B2G)
       Virtual   (B1G) (B3U) (AU) (AG) (B2U) (B1U) (B2G) (B3G)
                 (AG) (B1U) (B3G) (B1U) (AG) (B2U) (B3G) (B1U)
                 (B2U) (AG) (B3U) (B3G) (AG) (B2G) (B1U) (B1G)
                 (B2U) (B3U) (AU) (AG) (B2G) (B1U) (B3G) (AG) (B3G)
                 (B2U) (B1U) (B1U) (B2U) (AG) (B3G) (B1U) (AG)
                 (B1U) (B2U) (B3G) (AG) (B1U)
 The electronic state is 2-B1G.
 Alpha  occ. eigenvalues --  -11.44872 -11.44867 -11.44839 -11.44805 -11.39193
 Alpha  occ. eigenvalues --  -11.39175  -1.40581  -1.27105  -1.25661  -1.07738
 Alpha  occ. eigenvalues --   -1.05558  -0.95077  -0.87408  -0.86490  -0.82867
 Alpha  occ. eigenvalues --   -0.82346  -0.75886  -0.73477  -0.71774  -0.61572
 Alpha  occ. eigenvalues --   -0.56760
 Alpha virt. eigenvalues --   -0.14270  -0.05734   0.06299   0.12399   0.12737
 Alpha virt. eigenvalues --    0.13500   0.15369   0.18285   0.18392   0.25750
 Alpha virt. eigenvalues --    0.27380   0.29162   0.30225   0.56739   0.56948
 Alpha virt. eigenvalues --    0.66014   0.66704   0.70554   0.75124   0.78625
 Alpha virt. eigenvalues --    0.79353   0.82303   0.83569   0.84627   0.86834
 Alpha virt. eigenvalues --    0.89193   0.90550   0.94895   1.05620   1.07972
 Alpha virt. eigenvalues --    1.11926   1.17803   1.19114   1.20829   1.32440
 Alpha virt. eigenvalues --    1.33430   1.37848   1.49169   1.49816   1.51963
 Alpha virt. eigenvalues --    1.53550   1.57191   1.90507   1.92002   2.51096
  Beta  occ. eigenvalues --  -11.44043 -11.44037 -11.43995 -11.43963 -11.39907
  Beta  occ. eigenvalues --  -11.39890  -1.38150  -1.25982  -1.21592  -1.05316
  Beta  occ. eigenvalues --   -1.04842  -0.94632  -0.86613  -0.85826  -0.82616
  Beta  occ. eigenvalues --   -0.81728  -0.73250  -0.71293  -0.67936  -0.58500
  Beta virt. eigenvalues --   -0.24700  -0.04073  -0.02748   0.06625   0.13244
  Beta virt. eigenvalues --    0.13508   0.16217   0.16927   0.18287   0.18882
  Beta virt. eigenvalues --    0.26814   0.27996   0.29216   0.30620   0.57318
  Beta virt. eigenvalues --    0.57989   0.66325   0.66979   0.73695   0.75541
  Beta virt. eigenvalues --    0.79171   0.82413   0.83904   0.84692   0.87467
  Beta virt. eigenvalues --    0.88473   0.89106   0.90953   0.96026   1.05915
  Beta virt. eigenvalues --    1.08454   1.11820   1.17940   1.19518   1.20773
  Beta virt. eigenvalues --    1.32542   1.34030   1.38495   1.49868   1.50273
  Beta virt. eigenvalues --    1.52021   1.53187   1.59611   1.91834   1.92443
  Beta virt. eigenvalues --    2.51703
          Condensed to atoms (all electrons):
              1          2          3          4          5          6
     1  H    0.323171   0.383350  -0.023309  -0.000798   0.001362  -0.000013
     2  C    0.383350   5.252178   0.430973  -0.022162  -0.107640   0.001335
     3  C   -0.023309   0.430973   5.252331   0.373765   0.346517  -0.024649
     4  H   -0.000798  -0.022162   0.373765   0.317271  -0.024649  -0.000467
     5  C    0.001362  -0.107640   0.346517  -0.024649   5.252331   0.373765
     6  H   -0.000013   0.001335  -0.024649  -0.000467   0.373765   0.317271
     7  C   -0.000078   0.017042  -0.107640   0.001335   0.430973  -0.022162
     8  H    0.000000  -0.000078   0.001362  -0.000013  -0.023309  -0.000798
     9  C    0.001362  -0.107640   0.011038  -0.000058  -0.117240   0.001745
    10  H   -0.000013   0.001335  -0.000058   0.000000   0.001745  -0.000017
    11  C   -0.023309   0.430973  -0.117240   0.001745   0.011038  -0.000058
    12  H   -0.000798  -0.022162   0.001745  -0.000017  -0.000058   0.000000
              7          8          9         10         11         12
     1  H   -0.000078   0.000000   0.001362  -0.000013  -0.023309  -0.000798
     2  C    0.017042  -0.000078  -0.107640   0.001335   0.430973  -0.022162
     3  C   -0.107640   0.001362   0.011038  -0.000058  -0.117240   0.001745
     4  H    0.001335  -0.000013  -0.000058   0.000000   0.001745  -0.000017
     5  C    0.430973  -0.023309  -0.117240   0.001745   0.011038  -0.000058
     6  H   -0.022162  -0.000798   0.001745  -0.000017  -0.000058   0.000000
     7  C    5.252178   0.383350   0.430973  -0.022162  -0.107640   0.001335
     8  H    0.383350   0.323171  -0.023309  -0.000798   0.001362  -0.000013
     9  C    0.430973  -0.023309   5.252331   0.373765   0.346517  -0.024649
    10  H   -0.022162  -0.000798   0.373765   0.317271  -0.024649  -0.000467
    11  C   -0.107640   0.001362   0.346517  -0.024649   5.252331   0.373765
    12  H    0.001335  -0.000013  -0.024649  -0.000467   0.373765   0.317271
 Mulliken atomic charges:
              1
     1  H    0.339075
     2  C   -0.257504
     3  C   -0.144834
     4  H    0.354048
     5  C   -0.144834
     6  H    0.354048
     7  C   -0.257504
     8  H    0.339075
     9  C   -0.144834
    10  H    0.354048
    11  C   -0.144834
    12  H    0.354048
 Sum of Mulliken charges=   1.00000
 Atomic charges with hydrogens summed into heavy atoms:
              1
     1  H    0.000000
     2  C    0.081571
     3  C    0.209215
     4  H    0.000000
     5  C    0.209215
     6  H    0.000000
     7  C    0.081571
     8  H    0.000000
     9  C    0.209215
    10  H    0.000000
    11  C    0.209215
    12  H    0.000000
 Sum of Mulliken charges=   1.00000
          Atomic-Atomic Spin Densities.
              1          2          3          4          5          6
     1  H    0.018140   0.006809  -0.001452   0.000017   0.000061   0.000001
     2  C    0.006809  -0.430404  -0.000094   0.001654  -0.008483  -0.000053
     3  C   -0.001452  -0.000094   0.499405  -0.008708   0.024537  -0.001196
     4  H    0.000017   0.001654  -0.008708  -0.026085  -0.001196   0.000014
     5  C    0.000061  -0.008483   0.024537  -0.001196   0.499405  -0.008708
     6  H    0.000001  -0.000053  -0.001196   0.000014  -0.008708  -0.026085
     7  C    0.000021   0.007568  -0.008483  -0.000053  -0.000094   0.001654
     8  H    0.000000   0.000021   0.000061   0.000001  -0.001452   0.000017
     9  C    0.000061  -0.008483  -0.006925  -0.000016  -0.008842  -0.000025
    10  H    0.000001  -0.000053  -0.000016   0.000000  -0.000025   0.000002
    11  C   -0.001452  -0.000094  -0.008842  -0.000025  -0.006925  -0.000016
    12  H    0.000017   0.001654  -0.000025   0.000002  -0.000016   0.000000
              7          8          9         10         11         12
     1  H    0.000021   0.000000   0.000061   0.000001  -0.001452   0.000017
     2  C    0.007568   0.000021  -0.008483  -0.000053  -0.000094   0.001654
     3  C   -0.008483   0.000061  -0.006925  -0.000016  -0.008842  -0.000025
     4  H   -0.000053   0.000001  -0.000016   0.000000  -0.000025   0.000002
     5  C   -0.000094  -0.001452  -0.008842  -0.000025  -0.006925  -0.000016
     6  H    0.001654   0.000017  -0.000025   0.000002  -0.000016   0.000000
     7  C   -0.430404   0.006809  -0.000094   0.001654  -0.008483  -0.000053
     8  H    0.006809   0.018140  -0.001452   0.000017   0.000061   0.000001
     9  C   -0.000094  -0.001452   0.499405  -0.008708   0.024537  -0.001196
    10  H    0.001654   0.000017  -0.008708  -0.026085  -0.001196   0.000014
    11  C   -0.008483   0.000061   0.024537  -0.001196   0.499405  -0.008708
    12  H   -0.000053   0.000001  -0.001196   0.000014  -0.008708  -0.026085
 Mulliken atomic spin densities:
              1
     1  H    0.022223
     2  C   -0.429959
     3  C    0.488262
     4  H   -0.034394
     5  C    0.488262
     6  H   -0.034394
     7  C   -0.429959
     8  H    0.022223
     9  C    0.488262
    10  H   -0.034394
    11  C    0.488262
    12  H   -0.034394
 Sum of Mulliken spin densities=   1.00000
 Electronic spatial extent (au):  <R**2>=   442.7071
 Charge=     1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=     0.0000    Y=     0.0000    Z=     0.0000  Tot=     0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=   -37.3503   YY=   -18.6488   ZZ=   -22.5403
   XY=     0.0000   XZ=     0.0000   YZ=     0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=   -11.1705   YY=     7.5310   ZZ=     3.6395
   XY=     0.0000   XZ=     0.0000   YZ=     0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=     0.0000  YYY=     0.0000  ZZZ=     0.0000  XYY=     0.0000
  XXY=     0.0000  XXZ=     0.0000  XZZ=     0.0000  YZZ=     0.0000
  YYZ=     0.0000  XYZ=     0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=   -38.3063 YYYY=  -191.8078 ZZZZ=  -221.3304 XXXY=     0.0000
 XXXZ=     0.0000 YYYX=     0.0000 YYYZ=     0.0000 ZZZX=     0.0000
 ZZZY=     0.0000 XXYY=   -53.1240 XXZZ=   -60.1412 YYZZ=   -67.3031
 XXYZ=     0.0000 YYXZ=     0.0000 ZZXY=     0.0000
 N-N= 2.029039403459D+02 E-N=-9.288208059721D+02  KE= 2.286797203669D+02
 Symmetry AG   KE= 7.401472982072D+01
 Symmetry B1G  KE= 1.082696416029D+00
 Symmetry B2G  KE= 2.119117296348D+00
 Symmetry B3G  KE= 3.759984297920D+01
 Symmetry AU   KE= 3.590526156304D-17
 Symmetry B1U  KE= 7.180032874109D+01
 Symmetry B2U  KE= 4.027232310221D+01
 Symmetry B3U  KE= 1.790682011253D+00
 1|1|UNPC-UNK|SP|UTD-HF-FC|3-21G|C6H6(1+,2)|PCUSER|20-May-2009|0||# UHF
 /3-21G guess=read geom=allcheck td||_Benzene||1,2|H,0,0.,0.,-0.0113337
 391|C,0,0.,0.,1.0581929106|C,0,1.1879227257,-0.0000000001,1.7618895135
 |H,0,2.1278341857,0.,1.2473173325|C,0,1.1879227257,0.,3.2097104865|H,0
 ,2.1278341857,0.,3.7242826675|C,0,0.,-0.0000000001,3.9134070894|H,0,0.
 ,0.,4.9829337391|C,0,-1.1879227257,0.0000000001,3.2097104865|H,0,-2.12
 78341857,0.,3.7242826675|C,0,-1.1879227257,0.,1.7618895135|H,0,-2.1278
 341857,0.,1.2473173325||Version=IA32W-G03RevE.01|State=2-B1G|HF=-229.1
 274237|S2=0.859846|S2-1=0.|S2A=0.756774|RMSD=2.088e-010|Thermal=0.|PG=
 D02H [C2"(H1C1.C1H1),SG(C4H4)]||@


 THE ATTENTION SPAN OF A COMPUTER IS AS LONG
 AS ITS EXTENSION CORD.
 Job cpu time:  0 days  0 hours  0 minutes  9.0 seconds.
 File lengths (MBytes):  RWF=     24 Int=      0 D2E=      0 Chk=     12 Scr=      1
 Normal termination of Gaussian 03 at Wed May 20 12:30:43 2009.
